README.md

```diff
@@ -10,27 +10,38 @@
   <h1 align="center">Apostrophe Code Upgrader: A2 ➡ A3</h1>
 </p>
 
-The Code Upgrader handles the majority of required modifications for an Apostrophe 2 (A2) codebase to run Apostrophe 3 (A3). It will also identify many specific lines and sections of code that a developer will need to convert manually.
+The Code Upgrader handles a portion of the required modifications for an Apostrophe 2 (A2) codebase to run Apostrophe 3 (A3). It will also identify many specific lines and sections of code that a developer will need to convert manually.
 
 **Status:** In development (not for production use)
 
 ## Purpose
 
 ### What it does
 
+#### The `lint` command
```

Comment:
This is meant to be an overview of the tool as a whole. The lint command material should be at the start of  the "Linting the A2 codebase" section.